GNUmakefile

```diff
@@ -12,6 +12,8 @@ test: fmtcheck
 		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
+	@echo "Make sure nomad is running"
+	@echo "	nomad agent -dev -acl-enabled"
```

Comment:
As a rule of thumb, we tend to have the environment setup required for our acceptance tests happen outside the `testacc` target. This is so our CI and contributors have the option to reuse stable environments, and to avoid hardcoding the environment in.

Reply:
If I understood you correctly, you're suggesting, that `testacc` should depend on, say, `run_nomad`, which will start it for the user, correct?

If that's the case, I totally agree with you, but I just kept original approach.

Please confirm if I udnerstood you correctly, and I'll fix.

Reply:
Basically what I'm suggesting is that `testacc` not try to start Nomad at all. Instead, the user running `testacc` should start Nomad themselves, _then_ run `testacc`. A Makefile target can be added to run Nomad, but I think "running the tests" and "setting up the environment the tests run against" should be two separate commands.

Reply:
Oh I see, so it's the opposite of what I understood :)

Well, this does not setup the environment, but just merely a reminder for the user. But now that you pointed this out, I think, that a right place for this is README.md.

I'm going to address all this today.